docker/Dockerfile.cpu

```diff
@@ -16,16 +16,22 @@ RUN apt-get update && \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py
 
-RUN pip install mxnet-model-server
+RUN pip install mxnet-model-server \
+    && pip install mxnet-mkl
+
+RUN useradd -m model-server \
+    && mkdir -p /home/model-server/tmp \
+    && chown -R model-server /home/model-server
 
 COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
+COPY config.properties /home/model-server
+
 RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
 
-RUN useradd -m model-server
 USER model-server
 WORKDIR /home/model-server
-
-LABEL maintainer="dantu@amazon.com, rakvas@amazon.com, lufen@amazon.com, dden@amazon.com"
-
+ENV TEMP=/home/model-server/tmp
```

Comment:
Is this OK? Would sagemaker overwrite TEMP to some other directory? Or anyone else for that matter... Would we be better off creating our own temp instead of using something that is default, to avoid TEMP namespace collision?

Reply:
True. Maybe this is not a concern right away and we can address it later, but my concern is if we use these standard ones, we might get into issues when customers want to use TEMP/TMP or TMPDIR as something that they want to track ... For now I think its OK to tell customers that MMS requires read/write/modify access for TEMPDIR